common/models/signer.json

```diff
@@ -20,14 +20,15 @@
       "default": false
     },
     "created_at": {
-      "type": "date"
+      "type": "date",
+      "defaultFn": "now"
     }
   },
   "scope": {
     "order": "name ASC",
     "limit": 100
   },
-  "hidden": ["id", "email"],
+  "hidden": ["id", "email", "created_at"],
```

Comment:
El connector in-memory quan crea un model *Signer* no li afegeix la propetat `created_at` si no s'assigna un valor mentre que el connector de postgres si que retornava un `null`